reactioncommerce/reaction
# refactor: rewrite `accounts/sendResetPasswordEmail` into a GraphQL mutation

Resolves parts of #5529
Impact: minor
Type: feat|refactor

## Issue
`accounts/sendResetPasswordEmail` was a meteor method which used `getGraphQLContextInMeteorMethod`.

## Solution
Rewrite the meteor method to be a `sendResetAccountPasswordEmail` GraphQL mutation, which in turn will remove usage of `getGraphQLContextInMeteorMethod`.

## Breaking changes
Any custom code which used the `accounts/sendResetPasswordEmail` meteor method will need to be updated to use the GraphQL mutation

## Testing
1. Set up the email settings inside Reaction Admin so you receive account related emails
1. Sign up for a Reaction account, with a valid email address.
1. Use the "Forgot password" form to reset your password
1. See that you get the email (you'll need to set up `storefrontUrls` in your shop settings in order to actually reset the password)
1. Enter an email address that isn't attached to an account
1. See you receive an error message
